### DISCUSSION TITLE
MiniMap ＋React Portal导致node 不渲染？

### DISCUSSION THREAD

**Discussion by ahuigo** [antvis/X6#3622] [Q&A]
MiniMap ＋React Portal看起来有问题，node 渲染被洗掉了，但是minimap 小地图是ok的。

复现 Demo: https://codesandbox.io/s/trusting-blackburn-thxctn?file=/App.tsx

![image](https://github.com/antvis/X6/assets/1781999/de963180-e526-416b-a320-85e80556c1d7)

**Comment by Superzzw1994** [comment]
同问，请问解决了吗？

**Reply by ahuigo** [reply]
还没有，看起是minimap 需要new graph，而现在封装的portal不支持多graph.
demo: https://codesandbox.io/s/trusting-blackburn-thxctn?file=/App.tsx
